terraform/github/terraform.tfvars.json

```
@@ -150,7 +150,8 @@
           "jovial",
           "m-bull",
           "sjpb",
-          "stackhpc-ci"
+          "sd109",
+          "stackhpc-ci",
```

Need to remove the trailing comma.
```suggestion
          "stackhpc-ci"
```